### DISCUSSION TITLE
ERGMs - How to include activitity and popularity meaures?

### DISCUSSION THREAD

**Discussion by ElenaOS97** [statnet/ergm#452] [Q&A]
**How can we include measures for activity and popularity if it's not with istar(2) and ostar(2)?** 

​I tried the following two models to include activity and popularity measures, but they don't work:

a) Model with istar(2), ostar(2) and m2star. 
b) Model with idegree(2), odegree(2) and m2star. 

**Also, do idegree1.5 and odegree1.5 measure popularity and activity, respectively?** 

As I understand it, it's just a sum of the in- and out-degrees over the different nodes -> So, I don't think that it tells us if some nodes are more central than others in the network...

**Comment by martinamorris** [comment]
Hi Elena,

If what you are looking for is a summary descriptive measure of which nodes are more central in your network, you probably want to use the descriptive tools in our [SNA package](https://cran.r-project.org/web/packages/sna/sna.pdf).

The ergm package is used to fit Exponential Family Random Graph Models, which is a different type of analysis.  The purpose of an `ergm` is to identify significant differences between your network and a bernoulli (simple) random graph.  ERGMs estimate the odds of a tie as a function of the configurations it creates (e.g., nodal degree, nodal attribute matching, triadic configuration summaries, etc).

What is the purpose of your analysis?  We can probably provide better help if we know that.

**Reply by ElenaOS97** [reply]
Hi Martina, 

I would like to implement an analysis similar to  the one included in  page  43 of the book by Lusher and Robins. The authors 
compute an ERGM including parameters for activity and popularity. I am aiming to do the same.  I would like to know if the odds of a tie are greater as a function of whether there is centrality in the network. 
The link for the book is:
https://www.cambridge.org/core/books/exponential-random-graph-models-for-social-networks/9296EE2B53CDEF9FE9E2E981E2FDB8A8
